### ISSUE TITLE
action buttons

### ISSUE THREAD

**Issue by moreiraeduardo** [BroadSoft-Xtended/Project-Chrome-Extension#9]
Hi All.
The three action icons in the lower part of the Dialer window allow to activate :+1: Remote Office
Call Forwarding Always
Do Not Disturb

But they do not allow to deactivate those features. To deactivate those features, I have to go to the Assistant Enterprise or to the web configuration page.

Is this correct or this is a malfunction ?

**Comment by broadsoftxtended** [comment]
The action icons in the lower part of the Dialer allow activating and deactivating the features. If the channel connection is lost to Xsi-Events, then the action icons turn a grey color. At that time, the application is trying to reconnect and the buttons will not work until reconnection is successful. You can view the status of the reconnection attempts if you view the application logging in Google Chrome under the Tool-> Extensions -> click on background.html (for the Broadsoft Xtended dialer)

**Comment by moreiraeduardo** [comment]
No, all three buttons are grey, but activation works, deactivation does not work. This happens always.

**Comment by broadsoftxtended** [comment]
Can you verify the following:
1. Are you working off of the master branch?  Can you try updating/refreshing and trying again? If all three buttons are always grey disabled then neither activation/deactivation should work. 
2. Can you let us know which server version you are connecting to?
3. Also, can you look at the logs from background.html and see if you are getting any errors? If so, can you copy/paste the error from the log.
